src/clients/cassandra/CassandraConnector.js

```diff
@@ -65,13 +62,14 @@ function executeBatchMutations(mutations) {
  */
 function executeQuery(query, params) { // eslint-disable-line no-unused-vars
```

Comment:
How is paging supported for larger datasets? https://github.com/datastax/nodejs-driver#paging Should support a `fetchSize` parameter as opposed to the default of 5K records.